### PR TITLE
Implementing setSpeed for android

### PR DESCRIPTION
**Pull request by twitwi**
This uses the feature of the andoid media player to implement setSpeed.

The patch from https://github.com/zmxv/react-native-sound/pull/162 was need to actually get the latest master running on android.

Some initial ref: http://stackoverflow.com/a/37094353/2297277

I'm new to the project and the react native world so let me know if something need to be reworked in this PR.

### CODE REVIEW

**Comment by benvium** [comment]
@twitwi Looks great - thank you. It looks like you've combined #162 in here, so if we merge this one we can close #162. 

Code changes look good - one small thing, can you update the feature matrix on the main readme? Then I'll merge this. Thanks!

**Comment by rogierslag** [comment]
Awesome, could this be merged and released? The current 0.10.0 cannot play any sound on Android since a exception is thrown where that should not happen for local files

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -44,17 +44,6 @@ public void prepare(final String fileName, final Integer key, final Callback cal
       callback.invoke(e);
       return;
     }
-    try {
-      player.prepare();
```
Comment (cparram) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: @benvium I wonder why remove this lines. Maybe for this i can't play audio on android devices.

Comment (cparram) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: @benvium I wonder why remove this lines. Maybe for this I can't play audio from remote url on android devices.

**Pull request merged**